src/node.cc

```diff
@@ -3517,6 +3523,8 @@ static void PrintHelp() {
          "  --inspect-brk[=[host:]port]\n"
          "                             activate inspector on host:port\n"
          "                             and break at start of user script\n"
+         "  --inspect-brk-node         will break in node's bootstrap code\n"
+         "                             (default: false)\n"
```

Comment:
Maybe we can document this somewhere in `doc/guide`?

Reply:
Yeah, this seems to fall along the same lines as `--expose-internals`... we don't document that one for similar reasons

Reply:
I'll remove the info from PrintHelp and take a look at adding this to `doc/guide`. Thanks

Reply:
I've not really been able to find a good place for this in any of the docs in `doc/guide`. @joyeecheung Do you have any suggestions where this might go? 

Would it be alright if it is not documented as that seems to be the case for `--expose-internals`?